### PR TITLE
Fix/interface

### PR DESCRIPTION
**Pull request by Mihran9991**

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -1,143 +1,137 @@
...
+    }
+  </style>
+</head>
+
+<body>
+  <div id="buttons">
+    <button id="render">View Verifications</button>
+    <div id="auth-actions">
+      <button id="auth-popup">SAML in Popup</button>
+      <button id="auth-window">SAML in Separate Tab</button>
```
Comment (mkim177) [inline examples/index.html:66]: remove references to these buttons as well

File: src/sdk.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import { IVerifyIQ } from './types/SDK.interface';
-import { AuthTypes } from './types/auth-types.enum';
```
Comment (mkim177) [inline src/sdk.ts:2]: @Mihran9991 , don't remove the API functions for setting the Auth type. Instead, let's keep the code but not expose this publicly so it defaults to the "Popup" auth type for the customer but we actually manage that using these API functions.

So, the customer should never have to know that we have support for different auth types, but we use this code to set the authtype.

File: src/types/SDK.interface.ts [hunk 3]
```diff
@@ -1,7 +1,4 @@
-import { AuthTypes } from './auth-types.enum';
-
 export interface IVerifyIQ {
-  setAuth(authType: AuthTypes): this;
```
Comment (mkim177) [inline src/types/SDK.interface.ts:4]: same here, let's keep this code but not expose to the end-user

File: src/sdk.ts (hunk 2 above)
Comment (mkim177) [reply src/sdk.ts:2]: we need the same for `collectedDocumentWebhookUrl`. We'll need to support this configuration in the future, so  keep all the code that supports this config, but don't make the config required or mention anything in the documentation

**Review by mkim177 — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -50,41 +49,25 @@ const viq = new VerifyIQ({
 <div id="verify-iq-root"></div>
 ```
 
-
 **3. Configure SAML Login Method**
```
Comment (mkim177) [inline README.md:52]: remove this header.

File: src/sdk.ts [hunk 5]
```diff
@@ -41,13 +42,16 @@ class VerifyIQ implements IVerifyIQ {
...
     invariant(!!options.authToken, 'authToken is required');
+    // invariant(!!options.authType, 'authType is required');
 
     this.renderer = new Renderer();
 
     if (options.url) {
       this.renderer.setUrl(options.url);
     }
 
+    this.setAuth(AuthTypes.Popup);
```
Comment (mkim177) [inline src/sdk.ts]: this is still being hardcoded - we should default to Popup unless a value is passed in as an SDK option

**Review by mkim177 — CHANGES_REQUESTED** [review]

File: src/sdk.ts (hunk 5 above)
Comment (Mihran9991) [reply src/sdk.ts]: @mkim177 please check this 
[commit](https://github.com/Informed/verifyiq-sdk/pull/19/commits/4accc8012d9ba003bedfbfe9829bedfc8b26f911)

**Review by mkim177 — APPROVED** [review]

**Pull request merged**